homework-classes/ContributorsView.js

```diff
@@ -19,8 +19,46 @@
      * @param {Object[]} contributors An array of contributor objects
      */
     render(contributors) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('ContributorsView', contributors);
+      const contributorsContainer = document.querySelector(
```

Comment:
This container is already available to this class in the `this.container` class member. Therefore don't try and get it with `document.querySelector`.